pkg/build/nodeimage/internal/kube/builder_remote.go

```diff
@@ -0,0 +1,203 @@
+/*
```

Comment:
Let's call this one "release" ?

"remote" sort of implies there's a build happening on a remote machine, "release" implies we're using an upstream release. I think. Maybe there's an even better name.

Reply:
Or maybe "download" if that's too ambiguous versus "tarball"

Reply:
see consolidated table below - https://github.com/kubernetes-sigs/kind/pull/3614#issuecomment-2121474989